### PR TITLE
Resolve lasting problems with FHIR tests

### PR DESCRIPTION
**Pull request by mkralik3**
Resolved problems from this PR https://github.com/syndesisio/syndesis-qe/pull/1533

* Improve FHIR#isReady and FHIR#udeploy method, change FHIR server name
* Make FhirClientManager singleton
* Add hook for undeploy FHIR and close PORT FORWARD
* Add workaround for Patch operation

//test: `@fhir`

### CODE REVIEW

File: ui-tests/src/test/java/io/syndesis/qe/hooks/CommonHooks.java [hunk 1]
```diff
@@ -70,4 +72,11 @@ public void cleanDv() {
         log.info("Deleting DV addon");
         ResourceFactory.destroy(DV.class);
     }
+
+    @After
+    public void cleanFHIR() {
+        log.info("Removing FHIR");
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/hooks/CommonHooks.java:78]: this will be printed after each test

Comment (mkralik3) [reply ui-tests/src/test/java/io/syndesis/qe/hooks/CommonHooks.java:78]: :(, missing `@fhir`

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - DB to DB | Read and update operations 
```

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**